tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -11138,22 +11139,6 @@ bool Sema::DiagnoseHLSLMethodCall(const CXXMethodDecl *MD, SourceLocation Loc) {
         Diags.Report(Loc, diag::err_hlsl_wg_nodetrackrwinputsharing_missing);
         return true;
       }
-    } else if (opCode == hlsl::IntrinsicOp::MOP_CalculateLevelOfDetail ||
```

Comment:
I would think the call-site of this function, wherever it's called, would need to be updated to account for the case of the opcode being one of these two. The logic has moved, so wherever DiagnoseHLSLMethodCall is called, wouldn't we also need to call isIllegalIntrinsic now?

Reply:
Removed isIllegalIntrinsic.